docs/example.ts

```diff
@@ -86,191 +131,329 @@ async function setup(): Promise<{
 
   // ------------------------- Claimer  ----------------------------------------
   // How to generate an Identity
-  // const mnemonic = Kilt.Identity.generateMnemonic()
-  const claimer = Kilt.Identity.buildFromMnemonic(
+  // const mnemonic = Kilt.Utils.UUID.generate()
+  const claimerMnemonic =
     'wish rather clinic rather connect culture frown like quote effort cart faculty'
-  )
-  // const address = claimer.address
+
+  // Create a light DID from the generated authentication key.
```

Comment:
```suggest
// Generate authentication and encryption keys and then derive a light DID from them.
```

Or something like that.

Reply:
Ok, sounds better. I have added the following:

`// Generate authentication and encryption keys used to derive a light DID from them.`